builder-tools/extract_source.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# Standalone script to extract files from a Docker image.
```

Comment:
Can you add an Apache 2.0 license header here?

Reply:
Done.